app/models/user.rb

```diff
@@ -1,17 +1,22 @@
 class User < ApplicationRecord
-  # Association
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Please remove commented lines of code as they make your code base cluttered and difficult to read 👍